locksmith/migrations/20190318152733-create-user.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('Users', {
+      publicKey: {
+        type: Sequelize.STRING,
+        primaryKey: true,
+        allowNull: false,
+        unqiue: true
+      },
+      recoverySeed: {
+        type: Sequelize.STRING,
+        allowNull: false
+      },
+      encryptedPrivateKey: {
```

Comment:
Let's be more explicit and call it `passwordEncryptedPrivateKey`.

Reply:
Also, I believe we will use `web3.eth.accounts.encrypt(privateKey, password);` which yields a pretty complex object: https://web3js.readthedocs.io/en/1.0/web3-eth-accounts.html#encrypt

Is a string enough to store that?

Reply:
Interesting I was thinking that we would use something like HMAC256. Let's discuss the details so that we can get the storage to match our approach